packages/interpreter/src/ts/native.ts

```diff
@@ -277,11 +277,14 @@ export class NativeInterpreter extends JSChannel_ {
       bubbles,
     };
 
-    // Run any prevent defaults the user might've set
-    // This is to support the prevent_default: "onclick" attribute that dioxus has had for a while, but is not necessary
-    // now that we expose preventDefault to the virtualdom on desktop
-    // Liveview will still need to use this
-    this.preventDefaults(event);
+    // This should:
+    // - prevent form submissions from navigating
+    // - prevent anchor tags from navigating
+    // - prevent buttons from submitting forms
+    // - let the virtualdom attempt to prevent the event
+    if (event.type === "submit") {
+      event.preventDefault();
```

Comment:
to my understanding all submit events should be prevented here so the code below is not needed